docs/modules/configuring/proc-virtualkafkacluster-loadbalancer-dns-config.adoc

```diff
@@ -0,0 +1,55 @@
+
+// file included in the following:
+//
+// con-virtualkafkafkacluster-overview.adoc
+
+[id='proc-virtualkafkacluster-loadbalancer-dns-config-{context}']
+
+= Configuring DNS (load balancer ingress type only)
+
+[role="_abstract"]
+
+When using the `loadBalancer` ingress type, it is the responsibility of the user to ensure that both the `bootstrapAddress` and the names generated from `advertisedBrokerAddressPattern` resolve to the *external address* of the Kubernetes Service underlying the load balancer on the *network where the off-cluster applications run*.
+
+.Prerequisites
+
+* Kroxylicious Operator is installed.
+* `KafkaProxy`, `VirtualKafkaCluster`, and `KafkaProxyIngress` resources deployed.
+* `VirtualKafkaCluster`, and `KafkaProxyIngress` configured for use of a `loadBalancer`.
+* Ability to configure DNS on the network where the off-cluster applications run.
+
+.Procedure
+
+.  Run the following command to discover the *external address* being used by the load balancer.
++
+[source,shell]
+----
+kubectl get service -n <namespace> <proxy-name>-sni -o=jsonpath='{.status.loadBalancer.ingress[0].hostname}'
```

Comment:
some implementations will have an `ip` field not a `hostname`, maybe we should peel it back to `.status.loadBalancer.ingress[0]` which will display either of them:

minikube:
```
kubectl get service -nmy-proxy simple-sni -o=jsonpath='{.status.loadBalancer.ingress[0]}'
{"ip":"127.0.0.1","ipMode":"VIP"}
```

(on that note, minikube needs a tunnel before it has an ip set on the Service, I wonder if these docs should cater to the development k8s environments? I'm imagining tire kickers using a local env first)